scene/2d/tile_map.cpp

```diff
@@ -686,26 +686,30 @@ void TileMapLayer::_physics_update() {
 
 void TileMapLayer::_physics_notify_tilemap_change(TileMapLayer::DirtyFlags p_what) {
 	Transform2D gl_transform = tile_map_node->get_global_transform();
+	PhysicsServer2D *ps = PhysicsServer2D::get_singleton();
```

Comment:
Don't know that storing the server in a variable for three uses is necessary and makes the code harder to read IMO

Reply:
I tend to agree it's not necessary, but it doesn't seem to be a bad change either, so I'm fine merging as is (and mostly in a hurry to get the `master` branch somewhat RC1 ready ;)).

Reply:
> Don't know that storing the server in a variable for three uses is necessary and makes the code harder to read IMO (leftover from the original PR and was planning to comment the same there)

It's commonly done in the TileMap codebase (using `ps`, `rs` or `ns` for the different servers), so it's more or less consistent with what we already have I believe. I think anyone familiar with the TileMap-related code should be familiar with this, so it's does not hurt readability much I think. But yeah, that's debatable.